CHANGELOG.md

```diff
@@ -1,49 +1,42 @@
+* Support for endpoints added and changed endpoints changed after March 31, 2021
+* Support for multiple instances [issue #176](https://github.com/pipedrive/client-nodejs/issues/176)
+* Convenience functions regarding OAuth 2.0 lifecycle including `buildAuthorizationUrl`, `authorize` and `refreshToken`
+* Support for changing the host URL [issue #228](https://github.com/pipedrive/client-nodejs/issues/228)
+* Support for both snake_case and camelCase
+* `UpdateProductResponse` for `PUT /products/{id}`
+* `Team` (singular) schema for endpoints that deal with a single team
+* Added support for custom fields for `Content-Type: application/x-www-form-urlencoded`
+* Add into documentation that also a Lead is one of the possible returned entities in related objects for the search endpoint.
+* Add note field to call logs.
 ### Changed
 * Updated `MailThreadPut` schema to represent `object` in response instead of an `array`
 * Updated description for `MailThreadOne`
-### Removed
-* Deleted deprecated `note` field from all lead related documents.
-* Fixed typo in lead example response (`crrency` to `currency`)
-### Added
-* `UpdateProductResponse` for `PUT /products/{id}` 
-### Changed
 * Split `Product` schema into `BaseProduct` and `ProductWithArrayPrices` or `ProductWithObjectPrices` to represent `prices` property with `array` type and `object` type respectively
 * Use the new `ProductWithObjectPrices` in `UpdateProductResponse`
-### Added
-* `Team` (singular) schema for endpoints that deal with a signle team
-### Changed
 * `TeamSuccess` response schema to refer to `Team` (singular) instead of `Teams` (plural)
-### Changed
 * Extended `DELETE /calllogs/{id}` security with `api_key` property.
 * Extended `POST /callLogs/{id}/recordings` security with `api_key` property.
-##  [0.1.1] - 2021-06-02
-### Changed
 * Extended `GET /v1/roles/{id}/settings` response with `lead_default_visibility` property.
 * Extended `POST /v1/roles/{id}/settings` request payload with  `lead_default_visibility` property.
-### Changed
 * Deleted `MailThreads` and `MailMessages` from API operation groups and combined them under `Mailbox` group.
 * Extracted `LeadLabels` and `LeadSources` API operation groups from `Leads` group.
-### Added
-* Added support for custom fields for `Content-Type: application/x-www-form-urlencoded`
-### Deleted
-* Deleted unused files (copied latest auto-generated version)
-### Changed
 * Deleted `MailThreads` and `MailMessages` from API operation groups and combined them under `Mailbox` group.
 * Extracted `LeadLabels` and `LeadSources` API operation groups from `Leads` group.
-### Added
-* Add note field to call logs.
-### Added
-* Add into documentation that also a Lead is one of the possible returned entities in related objects for the search endpoint.
+### Removed
+* Deleted deprecated `note` field from all lead related documents.
+* Fixed typo in lead example response (`crrency` to `currency`)
```

Comment:
What do you think about moving it to the **Changed** section, or create **Fixed** one